log/audit-logger_test.go

```diff
@@ -83,10 +110,14 @@ func TestSyslogMethods(t *testing.T) {
 	audit.Audit("audit-logger_test.go: audit-notice")
 	audit.Crit("audit-logger_test.go: critical")
 	audit.Debug("audit-logger_test.go: debug")
-	// Don't test Emerg... it sends a wall to the host OS.
-	// audit.Emerg("audit-logger_test.go: emerg")
+	audit.Emerg("audit-logger_test.go: emerg")
 	audit.Err("audit-logger_test.go: err")
 	audit.Info("audit-logger_test.go: info")
 	audit.Notice("audit-logger_test.go: notice")
 	audit.Warning("audit-logger_test.go: warning")
+	audit.Alert("audit-logger_test.go: alert")
+}
+
+func TestEmergSyslog(t *testing.T) {
```

Comment:
Oops. Added that before changing to UDP so that Emerg wouldn't run `wall` on the system anymore. Realized I could just fix the whole thing. :)